src/tigerbeetle-java/src/main/java/com/tigerbeetle/JNILoader.java

```diff
@@ -19,7 +19,7 @@ public static OS getOS() {
             if (osName.startsWith("win")) {
                 return OS.win;
             } else if (osName.startsWith("macos") || osName.startsWith("osx")
```

Comment:
Thank you for catching that @ianmorgan
I believe this condition can be entirely reworked, it looks like [no Mac OS version reports itself](https://github.com/apache/commons-lang/blob/aebebf784bdc210073feabdd73a9730edcde20d7/src/main/java/org/apache/commons/lang3/SystemUtils.java#L1095-L1293) as `macos` or `osx`.

Can you please change to that instead:
```suggestion
            } else if (osName.startsWith("mac") || osName.startsWith("darwin")) {
```